charts/redpanda/chart_test.go

```diff
@@ -748,7 +749,16 @@ func TestGoHelmEquivalence(t *testing.T) {
 		Enabled: ptr.To(false),
 	}
 
-	values.Console = &redpanda.PartialConsole{Enabled: ptr.To(false)}
+	values.Console = &console.PartialValues{
+		Enabled: ptr.To(true),
+		Secret: &console.PartialSecretConfig{
```

Comment:
Can you add a note about why the secret is set here? At first I thought it was just being copied but then I realized we need it to be consistent cross both helm and go invocations.

Reply:
In the first implementation secret was always generated. Now as License is added it makes sense to have JWT placeholder for stable tests.

Do you need a one line comment that states "For generating stable test result"?